ethereum/core/src/test/java/org/hyperledger/besu/ethereum/vm/EstimateGasOperationTracerTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.vm;
+
+import static org.mockito.Mockito.mock;
+
+import org.hyperledger.besu.ethereum.core.Gas;
+import org.hyperledger.besu.ethereum.core.MessageFrameTestFixture;
+import org.hyperledger.besu.ethereum.vm.OperationTracer.ExecuteOperation;
+import org.hyperledger.besu.ethereum.vm.ehalt.ExceptionalHaltException;
+import org.hyperledger.besu.ethereum.vm.operations.CallCodeOperation;
+import org.hyperledger.besu.ethereum.vm.operations.SStoreOperation;
+
+import java.util.Optional;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Test;
+
+public class EstimateGasOperationTracerTest {
+
+  private EstimateGasOperationTracer operationTracer;
+  private MessageFrameTestFixture messageFrameTestFixture;
+
+  @Before
+  public void setUp() {
+    operationTracer = new EstimateGasOperationTracer();
+    messageFrameTestFixture = new MessageFrameTestFixture();
+  }
+
+  @Test
+  public void shouldDetectChangeInDepthDuringExecution() throws ExceptionalHaltException {
+
+    final ExecuteOperation noExecutionOperation = mock(ExecuteOperation.class);
+
+    Assertions.assertThat(operationTracer.getMaxDepth()).isEqualTo(0);
```

Comment:
(Suggestion): For consistency i would suggest to import statically `assertThat` method.

Reply:
yes you are right I just changed